### PR TITLE
Update for OpenSpeedTest application demo

### PR DESCRIPTION
**Pull request by kfirtoledo**
Support new controlplane for the demos of OpenSpeedTest applications

### CODE REVIEW

File: demos/speedtest/kind/apply_policy.py [hunk 1]
```diff
@@ -12,65 +12,66 @@
...
+    gw = args["peer"]
     type = args["type"]
-    gwctlName = mbg[:-1]+"ctl"+ mbg[-1]
 
 
     print(f'Working directory {proj_dir}')
     os.chdir(proj_dir)
 
-    applyPolicy(mbg, gwctlName, type)
+    applyPolicy(gw, type)
```
Comment (elevran) [inline demos/speedtest/kind/apply_policy.py:76]: nit: github seems to mark this file as missing newline at the end?

File: demos/speedtest/testdata/manifests/speedtest.yaml [hunk 2]
```diff
@@ -45,4 +45,19 @@ spec:
...
+  labels:
+    app: openspeedtest
+    service: openspeedtest
+spec:
+  ports:
+  - port: 3000
+    targetPort: 3000
+    name: http
+  selector:
+    app: openspeedtest
```
Comment (elevran) [inline demos/speedtest/testdata/manifests/speedtest.yaml:63]: nit: ditto - missing newline at EOF

**Review by elevran — COMMENTED** [review]
Cursory review completed. My python skills are close to zero...

**Review by praveingk — APPROVED** [review]
The openspeedtest demo is for showing policies right. Would be good to mention what the demo actually demonstrates in the README.

**Pull request merged**